pledgeindia/pledge/views.py

```diff
@@ -1,3 +1,8 @@
 from django.shortcuts import render
-
+from django.http import JsonResponse
+import * from utils
 # Create your views here.
+
+def JsonData(request):
```

Comment:
Maybe name the view appropriately here ? @thisisayush @omi10859

Reply:
Right. How about `GetPledgesView(request)`?